### PR TITLE
feat/add-children-prop-to-breadcrumb

### LINKED ISSUE #2174: breadcrumb expand-collapse cannot become controlled
**Is your iteration request related to a problem?**
Do you need more flexibility in your theme? Please add a clear and concise description of what the current problem is.

**Describe the solution you'd like**
A clear and concise description of what you want to achieve.

**Additional context**

### PR DESCRIPTION
**Pull request by lolarufino**
### Types of changes

- [ ] 🪲 Bug fix (non-breaking change which fixes an issue)
- [x] ✨ New feature (non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 🧾 Documentation
- [ ] 📷 Demo
- [ ] 🧪 Test
- [ ] 🧠 Refactor
- [ ] 💄 Styles

### Description, Motivation and Context
Add children prop because we need it to pass a component

### Screenshots - Animations
image

### CODE REVIEW

**Review by tomasmax — APPROVED** [review]

File: components/molecule/breadcrumb/src/index.js [hunk 1]
```diff
@@ -20,17 +21,18 @@ const BreadcrumbBasic = ({
   return (
     <nav aria-label="breadcrumb" role="navigation">
       <div className={breadcrumbClassName({isExpanded, isScrollable})}>
-        <button onClick={expandBreadcrumb} className="sui-BreadcrumbBasic-btn">
+        {children && <div className={`${BASE_CLASS}-children`}>{children}</div>}
```
Comment (tomasmax) [inline components/molecule/breadcrumb/src/index.js]: maybe a more meaningful name instead of children? `leftElement` or `leftAddon` ?

**Review by javiauso — APPROVED** [review]

**Comment by andresin87** [comment]
Why you need an addon in the breadcrumb and not beside?

File: components/molecule/breadcrumb/src/index.js [hunk 2]
```diff
@@ -73,15 +93,25 @@ BreadcrumbBasic.propTypes = {
...
-  linkFactory: ({to, href, className, children}) => (
-    <a href={to || href} className={className}>
-      {children}
-    </a>
-  )
+  isScrollable: PropTypes.bool,
+  /**
+   * Element to render at the left
+   */
+  leftAddon: PropTypes.node,
```
Comment (tomasmax) [inline components/molecule/breadcrumb/src/index.js]: You can remove this prop

**Review by andresin87 — APPROVED** [review]

File: components/molecule/breadcrumb/src/index.js [hunk 3]
```diff
@@ -1,36 +1,61 @@
...
+  isScrollable = false,
+  isExpanded,
+  defaultIsExpanded = false,
+  onExpand,
+  onCollapse,
+  onClick
 }) => {
-  const [isExpanded, setIsExpanded] = useState(false)
-  const expandBreadcrumb = () => setIsExpanded(true)
+  const [isExpandedState, setIsExpandedState] = useControlledState(
```
Comment (andresin87) [inline components/molecule/breadcrumb/src/index.js:23]: @sergiqm94 @stefi23 @Pato2790 👀

**Pull request merged**